### DISCUSSION TITLE
Custom height

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [yitengjun/ukiyo-js#22] [Q&A] [answered]
Hello! Please suggest me how can I set a custom height? Using bootstrap I have height 100/ 75/50. Can I pass some how img element with your library into that div? Ty so much! 😊

**Comment by yitengjun** [comment, marked as answer]
If you're using Bootstrap, you can set a custom height using the following method:

```
<div class="row" style="height: 800px">
  <div class="col-lg-4 h-50">
    <img class="ukiyo w-100 h-100" src="~" />
  </div>
</div>
```

Please ensure that you're using version [v4.0.1](https://github.com/yitengjun/ukiyo-js/releases) or higher. Note that this library may not be fully compatible with Bootstrap, so use caution when using them together.

**Reply by ritchiemarkov** [reply]
Thank you so much for your kind answer. Ok i will give it a try!
